Cargo.toml

```diff
@@ -14,7 +14,10 @@ rust-version = "1.56"
 bench = false
 
 [features]
-default = ["ryu"]
+default = ["std", "ryu"]
+
+alloc = []
+std = ["alloc"]
 
 [dependencies]
```

Comment:
A bunch of dependencies pull in std by default. I think it's optional for most / all so you can just add `default-features = false` and conditionally re-enable any features if necessary.